liberty-maven-plugin/src/test/resources/checkErrorVersionlessFeature.bsh

```diff
@@ -9,6 +9,9 @@ switch (testname) {
     case "kernel-install-versionless-feature-fail-no-platform-it":
         expectedMessage = "CWWKF1516E: The platform could not be determined. The following versionless features cannot be installed: [servlet]";
          break;
+    case "springboot-3-deploy-multiple-springboot-application-nodes-fail-it":
```

Comment:
If we are going to add other tests to this script, we should rename it probably.